README.md

```diff
@@ -881,6 +881,29 @@ interface FCPAttribution {
 #### `INPAttribution`
 
 ```ts
+/**
+ * Summary information about the longest script intersecting the INP duration
+ * as provided by the Long Animation Frame API.
+ *
+ * NOTE: Only scripts above 5 milliseconds are included in long animation
+ * frames.
+ */
+interface LongestScriptSummary {
+  /**
+   * The longest Long Animation Frame script that intersects the INP
+   * interaction.
+   */
+  entry: PerformanceScriptTiming;
+  /**
+   * The INP sub-part where the longest script ran.
```

Comment:
Non-blocking: we should change these to subpart throughout now that we've standardized on that in our docs.

Reply:
Done.